src/main/java/core/basesyntax/FileWork.java

```diff
@@ -9,6 +15,26 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        if (fileName.isEmpty()) {
+            return new String[]{};
+        }
+        FileReader fr = null;
+        try {
+            fr = new FileReader(fileName);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+        Scanner scanner = new Scanner(fr);
+        ArrayList<String> sb = new ArrayList<>();
+        while (scanner.hasNext()) {
+            String word = scanner.next();
+            if (word.startsWith("W") || word.startsWith("w")) {
```

Comment:
```suggestion
            if (word.startsWith("W") || word.startsWith("w")) {
```
try to solve with a single check, don't use both

Reply:
and make a "w" as a constant